massa-client/src/cmds.rs

```diff
@@ -627,16 +628,14 @@ impl Command {
                 for key in parse_vec::<Address>(parameters)?.into_iter() {
                     match wallet.remove_address(key) {
                         Ok(_) => {
-                            res.push_str(&format!("Removed address {} from the wallet\n", key));
+                            let _ = writeln!(res, "Removed address {} from the wallet", key);
```

Comment:
Why the `let _ =` ?

Reply:
Because we don't use the result. It's the recommended practice by clippy here : https://rust-lang.github.io/rust-clippy/master/index.html#format_push_string